### DISCUSSION TITLE
How do I set quote preference to 'single' for Svelte files?

### DISCUSSION THREAD

**Discussion by jamauro** [zed-industries/zed#6593] [Feature Requests]
I'd prefer to use single quotes but when I click save, Zed changes them to double quotes. This appears to only happen in `.svelte` files.

Here's what I tried inside of Zed settings to no avail:

```
"lsp": {
    "typescript-language-server": {
      "initialization_options": {
        "preferences": {
          "quotePreference": "single"
        }
      }
    }
  }
  ```

**Comment by korywka** [comment]
I think it is not typescript server, but https://github.com/sveltejs/language-tools/blob/master/packages/language-server/README.md#sveltepluginsvelteformatconfigsinglequote

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
